api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -585,6 +585,13 @@ public static final Collection<String> AUTO_ROLES() {
 	public static final String GP_NEXT_ORDER_NUMBER_SEED = "order.nextOrderNumberSeed";
 	
 	public static final String GP_ORDER_NUMBER_GENERATOR_BEAN_ID = "order.orderNumberGeneratorBeanId";
+
+	/**
+	 *  @since 2.7.8, 2.8.2, 2.9.0
+	 */
+	public static final String GP_ALLOW_SETTING_ORDER_NUMBER = "order.allowManuallySettingOrderNumber";
+	
+	public static final String GP_IGNORE_ATTEMPTS_TO_STOP_INACTIVE_ORDERS = "order.ignoreAttemptsToStopInactiveOrders";
```

Comment:
I don't understand why this is named the way it is

Reply:
Yes, it's mouthful but not sure of a better name.  Thoughts? "allowStoppingInactiveOrders"?